Geometry/TrackerCommonData/plugins/dd4hep/DDCutTubsFromPoints.cc

```diff
@@ -159,7 +160,7 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
       n_y_t /= norm;
       n_z_t /= norm;
 
-      auto seg = dd4hep::CutTube(r_min, r_max, dz, phi1, phi2, n_x_l, n_y_l, n_z_l, n_x_t, n_y_t, n_z_t);
+      auto seg = dd4hep::CutTube(segname, r_min, r_max, dz, phi1, phi2, n_x_l, n_y_l, n_z_l, n_x_t, n_y_t, n_z_t);
```

Comment:
In the old DD version of this file, the corresponding line is:
```
DDSolid seg = DDSolidFactory::cuttubs(segname, dz, r_min, r_max, phi1, dphi, n_x_l, n_y_l, n_z_l, n_x_t, n_y_t, n_z_t);
```
It seems clear `segname` is intended to be used for the name of this Cut Tube. Making this change creates an exact match between old DD and DD4hep for these solids.